src/main/kotlin/org/web3j/evm/ConsoleDebugTracer.kt

```diff
@@ -675,7 +659,7 @@ open class ConsoleDebugTracer(protected val metaFile: File?, private val reader:
         } catch (ex: IOException) {
             val enumSet = EnumSet.allOf(ExceptionalHaltReason::class.java)
             enumSet.add(ExceptionalHaltReason.NONE)
```

Comment:
if we are throwing directly the `ExceptionalHaltReason.NONE`, why do we need the `enumSet` ?

Reply:
Thanks for pointing this out, Changed the ExceptionalHaltException file from .java to kotlin now removed these unwanted lines